wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -252,6 +252,24 @@ public static Command repeatingSequence(Command... commands) {
     return sequence(commands).repeatedly();
   }
 
+  /**
+   * Runs individual commands in a series without grouped behavior; once the last command ends, the series is restarted.
+   *
+   * <p>Each command is run independently by proxy. The requirements of
+   * each command are reserved only for the duration of that command and
+   * are not reserved for an entire group process as they are in a
+   * grouped sequence.
+   *
+   * @param commands the commands to include in the series
+   * @return the command to run the series of commands repeatedly
+   * @see #sequence(Command...) use sequenceRepeatedly() to invoke repeated group sequence behavior
```

Comment:
Is `sequence(Command...)` this the right method to link to?

Reply:
Changed it to a better reference.